.github/workflows/publish-release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           - jdk-version: 11
             dist: centos7
```

Comment:
There is also a [`11-ubi9-minimal` image for JDK 11](https://hub.docker.com/layers/library/eclipse-temurin/11-ubi9-minimal/images/sha256-6605803ab68fdda19a051c7ee4f9a5504eb4fcf48b9212c741ed2a62cf71d5cd?context=explore)

Would you mind updating your PR so that we also use this image for JDK 11 please?

Reply:
Added the changes